test/cmdlineTests.sh

```diff
@@ -130,8 +130,13 @@ test_solc_behaviour() {
     exitCode=$?
     set -e
 
-    sed -i -e '/^Warning: This is a pre-release compiler version, please do not use it in production./d' "$stderr_path"
-    sed -i -e 's/ Consider adding "pragma .*$//' "$stderr_path"
+    if [[ "$solc_args" == *"--standard-json"* ]]; then
+        sed -i -e 's/{[^{]*Warning: This is a pre-release compiler version[^}]*},\{0,1\}//' "$stdout_path"
+        sed -i -e 's/,"errors":\[\]//' "$stdout_path"
+    else
+        sed -i -e '/^Warning: This is a pre-release compiler version, please do not use it in production./d' "$stderr_path"
+        sed -i -e 's/ Consider adding "pragma .*$//' "$stderr_path"
```

Comment:
Hm, what about this one?

Reply:
It would probably be better to remove it and to add ``pragma solidity >=0.0.0;`` to the all test contracts "*.sol" - it's in all json tests, though, so we don't need to add it for the standard json case.

Reply:
Would be noticeable on both release and non-release branches, if that was a problem.